### DISCUSSION TITLE
Searching for names for a github organization around vcpkg stuff

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#36189] [General]
Due to https://github.com/Neumann-A/my-vcpkg-ports/issues/2 I am searching for names for a github organization around stuff related to vcpkg. Current proposals are `vcpkg-contrib` and `open-vcpkg`. 

The idea is to have a place which can move a bit faster and experiment a bit more with languages outside of C/CXX. At the same time it should be a bit more official than my personal playgrounds https://github.com/Neumann-A/my-vcpkg-ports and https://github.com/Neumann-A/my-vcpkg-triplets. 
However, even though it is a bit more official at the end the idea is also to get those ports or changes into mainstream vcpkg. 

The idea here is to collect names for possible names and later have a vote on it. 

**Deadline to suggest names: 2024-01-28**

Pinging people who might be interested: @m-kuhn, @autoantwort , @dg0yt, @Osyotr, @cenit, @Thomas1664, @Hoikas

**Comment by Osyotr** [comment]
I like `open-vcpkg`.
But what about CI coverage in the new repo?

**Reply by autoantwort** [reply]
Maybe only test the changed port and the ports that depend on the port. I think I could provide some ci resources

**Reply by Neumann-A** [reply]
> But what about CI coverage in the new repo?

I think we'll figure something out. It is probably a lot easier to get community provided runners integrated. Secondly, we don't need full 2d testing as the main vcpkg repo does. Thirdly, we could still run PRs against main vcpkg since the goal is to get stuff integrated into the main registry, so at some point it has to be run against it anyway. The important thing here is to not flood it.

**Comment by dg0yt** [comment]
Shall we reserve the name before proposing?

**Reply by Neumann-A** [reply]
Although I don't think there are name stealing trolls around here I took `open-vcpkg` to make sure it stays this way. So do as you see fit.

**Comment by cenit** [comment]
i like your idea of a community-based org very much. And also exploiting vcpkg ci infrastructure using PR when needed :)
I was thinking of a portmanteau for the org name, something along the lines of openpkg, but that name in particular is already taken by another package manager, so it's out of discussion due to possible confusion...

**Comment by dg0yt** [comment]
Offering `vcpkg-hub`.

**Comment by m-kuhn** [comment]
I have the need for keeping some legacy ports alive (qt5 related), so I'd like to also run a registry that is not planned for upstream integration

**Comment by Thomas1664** [comment]
I like the idea of a community based organization. @autoantwort and myself were already thinking about having our own version of vcpkg-tool because the team doesn't seem to be interested in investing time for reviewing PRs.

**Comment by StarGate-One** [comment]
I think a community based version of the repos vcpkg and vcpkg tool would be an avenue worth exploring.

vcpkg-hub and open-vcpkg are both very good options.

I favor open-vcpkg myself.
